app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function should verify that the input string `command` contains exactly three elements separated by spaces, and that the first element is 'cp'. This is important to ensure the command is valid as per the checklist requirement.